src/java/org/apache/cassandra/cql3/conditions/ColumnCondition.java

```diff
@@ -294,6 +296,12 @@ public boolean appliesTo(Row row)
             return operator.isSatisfiedBy(column.type, rowValue(row), value);
         }
 
+        @Override
+        public boolean isNull(Row row)
+        {
+            return column.type.isNull(rowValue(row));
```

Comment:
`isNull` has the following semantics

1) for non-meangingless emptyness `null` and `empty` are both `null`
2) for meangingless emptyness `null` and `empty` are both `null` 
3) for types that support empty, only `null` is `null`

The main difference between 1/2 is that in 1 `empty` is rejected during validation, so normally means you passed a tombstone here by mistake (historic bug on the read path in `RowFilter`).  2 allows `empty` during validation